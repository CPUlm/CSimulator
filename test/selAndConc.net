INPUT
OUTPUT c
VAR
  a : 3, b, c : 6
IN
a = 011
b = SELECT 1 a
c = CONCAT b 00101

