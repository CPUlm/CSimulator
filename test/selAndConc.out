Step 1:
=> c = 100101
